### PR TITLE
initial scaffolding

### PR DESCRIPTION
**Pull request by bengerman13**
## Changes proposed in this pull request:

- add the basic tools and pipelines. Note that I've created this intending for a [`hello production`](https://blog.thepete.net/blog/2019/10/04/hello-production/) deployment model

## Security considerations

None

### CODE REVIEW

File: docker/Dockerfile.dev [hunk 1]
```diff
@@ -0,0 +1,91 @@
+FROM python:3.8-slim as base
```
Comment (ccostino) [inline docker/Dockerfile.dev:1]: Should we grab a new Python image at some point?

File: docker/Dockerfile.dev [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+# Install Fake Pebble LE server
+COPY --from=letsencrypt/pebble /usr/bin/pebble /usr/bin/pebble
+COPY --from=letsencrypt/pebble /test/ /test/
+COPY --from=letsencrypt/pebble-challtestsrv /usr/bin/pebble-challtestsrv /usr/bin/pebble-challtestsrv
+
+RUN cp /test/certs/pebble.minica.pem /usr/local/share/ca-certificates/pebble.crt \
+ && update-ca-certificates
+
+# Install Postgresql
+ENV PG_MAJOR=11
```
Comment (ccostino) [inline docker/Dockerfile.dev:44]: Grab a newer PostgreSQL version too at some point?

**Review by ccostino — APPROVED** [review]
Thanks, @bengerman13!  A couple of notes about updating Python and PostgreSQL versions, but I also realize that may involve additional testing of things that we don't necessarily have time for right now.

File: docker/Dockerfile.dev (hunk 2 above)
Comment (bengerman13) [reply docker/Dockerfile.dev:44]: no. we're testing against the version we have deployed

File: docker/Dockerfile.dev (hunk 1 above)
Comment (bengerman13) [reply docker/Dockerfile.dev:1]: maybe. I wouldn't mind features of 3.9, but 3.8 is plenty modern for now

**Pull request merged**